news/2018-12-31-project-loved-week-of-december-30th.md

```diff
@@ -0,0 +1,229 @@
+
+[![](/wiki/shared/news/2018-12-31-project-loved-week-of-december-30th/osu/kyoumei-no-true-force.jpg)](https://osu.ppy.sh/community/forums/topics/849038)
+
+[Harada Hitomi - Kyoumei no True Force](https://osu.ppy.sh/beatmapsets/315159#osu) by [Asphyxia](https://osu.ppy.sh/users/1715720)  
+*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*
+
+A true classic, Asphyxia's Kyoumei No True Force has weathered the storm and continues to this day to be a clean yet testing map that nearly everyone has seen or played before. Being mostly aim focused, its star rating is deceiving, and with the difficulty creeping up as the map plays out you might find yourself pushing the limits of your abilities as you near the marathon's end. With 70 favourites, I'm sure you'll enjoy it too, regardless of whether you're playing it for the first or even the tenth time.
+
+[![](/wiki/shared/news/2018-12-31-project-loved-week-of-december-30th/osu/mikusabbath.jpg)](https://osu.ppy.sh/community/forums/topics/849039)
+
+[Utsu-P - MiKUSABBATH](https://osu.ppy.sh/beatmapsets/264483#osu) by [Kuron-kun](https://osu.ppy.sh/users/2697284) and [Tarrasky](https://osu.ppy.sh/users/4098393)  
+*written by [Bubbleman](https://osu.ppy.sh/users/5182050)*
+
+A tribute to older styles of maps, MiKUSABBATH's style is clean, yet it's easy to find patterns inspired by older creations. Squares are scattered throughout, and with few streams this is another aim test yet with a different feel and flow. A lower difficulty by Tarrasky is included with an even older style yet still producing the same nostalgic feeling, and when combined with this banger of a song, the set produces a feeling best appreciated by experiencing it yourself. What are you waiting for? Give it a go!
+
+[![](/wiki/shared/news/2018-12-31-project-loved-week-of-december-30th/osu/all-i-need-are-things-i-like.jpg)](https://osu.ppy.sh/community/forums/topics/849040)
+
+[PinocchioP - All I Need are Things I Like](https://osu.ppy.sh/beatmapsets/537768#osu) by [Nattu](https://osu.ppy.sh/users/1886722)  
+*written by [BeasttrollMC](https://osu.ppy.sh/users/3171691)*
+
+Those who follow Project Loved will recognize this title from a previous map covered by "nameless" that was Loved back in November. This time we have the original by PinocchioP. Dating back to 2016, this map by Nattu was the first to gain the song popularity throughout the community with help from being featured in HappyStick's Summer Tour map pool. With over 120 favorites, it made a name for itself as a challenging alternating styled map for players to practice on while listening to some good tunes.
+
+[![](/wiki/shared/news/2018-12-31-project-loved-week-of-december-30th/osu/genesis-of-aquarion.jpg)](https://osu.ppy.sh/community/forums/topics/849041)
+
+[Sousei no Aquarion - Genesis of Aquarion](https://osu.ppy.sh/beatmapsets/637097#osu) by [Musty](https://osu.ppy.sh/users/251683)  
```

Comment:
this put the correct title as the artist and kept their current title.. whups.
chum - Sousei no Aquarion